### PR TITLE
OpenIddict.Pro Module documentation

### PR DESCRIPTION
**Pull request by gterdem**
Resolves a step of https://github.com/volosoft/volo/issues/10329

<strike>Waiting for https://github.com/volosoft/volo/issues/11607</strike> will be considered later.

https://github.com/abpio/abp-commercial-docs/blob/gterdem/openiddict_module_docs/en/modules/openiddict.md

### CODE REVIEW

File: en/modules/openiddict.md [hunk 1]
```diff
@@ -0,0 +1,462 @@
+# OpenIddict Module
+
+This module provides integration and management functionality for Identity Server;
```
Comment (hikalkan) [inline en/modules/openiddict.md]:
```suggestion
This module provides integration and management functionality for the OpenIddict library;
```

Also, give link for who doesn't know it.

File: en/modules/openiddict.md [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+
+This module follows the [module development best practices guide](https://docs.abp.io/en/abp/latest/Best-Practices/Index) and consists of several NuGet and NPM packages. See the guide if you want to understand the packages and relations between them.
+
+You can visit [Identity module package list page](https://abp.io/packages?moduleName=Volo.Identity.Pro) to see list of packages related with this module.
+
+## User Interface
+
+### Menu Items
+
+Identity Server module adds the following items to the "Main" menu, under the "Administration" menu item:
```
Comment (hikalkan) [inline en/modules/openiddict.md]:
```suggestion
The OpenIddict module adds the following items to the "Main" menu, under the "Administration" menu item:
```

File: en/modules/openiddict.md [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+* Built on the [OpenIddict-core](https://github.com/openiddict/openiddict-core) library.
+* Manage **Application** and **API resources** in the system. 
+* Set **permissions** for clients.
+* Manage **API resources**
+
+See [the module description page](https://commercial.abp.io/modules/Volo.openiddict.Ui) for an overview of the module features.
+
+## How to Install
+
+OpenIddict is pre-installed in [the startup templates](../Startup-Templates/Index) from version 6.0.0-rc1. So, no need to manually install it.
```
Comment (hikalkan) [inline en/modules/openiddict.md]: Give the migration link to users who were using a previous version.

File: en/modules/openiddict.md [hunk 4]
```diff
@@ -0,0 +1,462 @@
+# OpenIddict Module
+
+This module provides integration and management functionality for Identity Server;
+
+* Built on the [OpenIddict-core](https://github.com/openiddict/openiddict-core) library.
+* Manage **Application** and **API resources** in the system. 
+* Set **permissions** for clients.
+* Manage **API resources**
+
+See [the module description page](https://commercial.abp.io/modules/Volo.openiddict.Ui) for an overview of the module features.
```
Comment (hikalkan) [inline en/modules/openiddict.md]: Link should be https://commercial.abp.io/modules/Volo.OpenIddict

File: en/modules/openiddict.md [hunk 5]
```diff
@@ -0,0 +1,462 @@
+# OpenIddict Module
+
+This module provides integration and management functionality for Identity Server;
+
+* Built on the [OpenIddict-core](https://github.com/openiddict/openiddict-core) library.
+* Manage **Application** and **API resources** in the system. 
+* Set **permissions** for clients.
+* Manage **API resources**
```
Comment (hikalkan) [inline en/modules/openiddict.md]: Not managing API resources as I know. Also, it is duplicated (second item also says it)

File: en/modules/openiddict.md [hunk 6]
```diff
@@ -0,0 +1,462 @@
+# OpenIddict Module
+
+This module provides integration and management functionality for Identity Server;
+
+* Built on the [OpenIddict-core](https://github.com/openiddict/openiddict-core) library.
+* Manage **Application** and **API resources** in the system.
```
Comment (hikalkan) [inline en/modules/openiddict.md]:
```suggestion
* Manage **Application** and **Scopes** in the system. 
```

File: en/modules/openiddict.md [hunk 7]
```diff
@@ -0,0 +1,462 @@
...
+
+## How to Install
+
+OpenIddict is pre-installed in [the startup templates](../Startup-Templates/Index) from version 6.0.0-rc1. So, no need to manually install it.
+
+## Packages
+
+This module follows the [module development best practices guide](https://docs.abp.io/en/abp/latest/Best-Practices/Index) and consists of several NuGet and NPM packages. See the guide if you want to understand the packages and relations between them.
+
+You can visit [Identity module package list page](https://abp.io/packages?moduleName=Volo.Identity.Pro) to see list of packages related with this module.
```
Comment (hikalkan) [inline en/modules/openiddict.md:19]: Text and link are wrong. This is not Identity module.

File: en/modules/openiddict.md [hunk 8]
```diff
@@ -0,0 +1,462 @@
...
+
+## User Interface
+
+### Menu Items
+
+Identity Server module adds the following items to the "Main" menu, under the "Administration" menu item:
+
+* **Applications**: Application management page.
+* **Scopes**: API resource management page.
```
Comment (hikalkan) [inline en/modules/openiddict.md]:
```suggestion
* **Scopes**: Scope management page.
```

**Review by hikalkan — CHANGES_REQUESTED** [review]

**Comment by hikalkan** [comment]
@maliming I didn't review deeply. It would be better you review this. You can merge if it is OK (and if my comments are fixed).

File: en/modules/openiddict.md (hunk 7 above)
Comment (gterdem) [reply en/modules/openiddict.md:19]: I thought about the link between Identity module.
IdentityServer docs has the same info: 
https://github.com/abpio/abp-commercial-docs/blob/4ce1975088eefe3ebfc9d8d7efaf3d8b3e6c7b35/en/modules/identity-server.md?plain=1#L22

File: en/modules/openiddict.md (hunk 1 above)
Comment (gterdem) [reply en/modules/openiddict.md]: The next line has the link, shall we add link anyway?

https://github.com/abpio/abp-commercial-docs/blob/c211203d84f67da96907debc4bc117df7285064a/en/modules/openiddict.md?plain=1#L5

File: en/modules/openiddict.md (hunk 3 above)
Comment (gterdem) [reply en/modules/openiddict.md]: https://github.com/abpio/abp-commercial-docs/blob/c211203d84f67da96907debc4bc117df7285064a/en/modules/openiddict.md?plain=1#L13

**Pull request merged**